src/AuthorizedSocketConnection.js

```diff
@@ -176,6 +176,7 @@ export default class AuthorizedSocketConnection<TContext, TCredentials> {
         document,
         null,
         {
+          credentials: this.config.credentialsManager.getCredentials(),
```

Comment:
this doesn't quite work, though – this gets evaluated once statically, but the actual value can change

Reply:
what about `credentialsManager: this.config.credentialsManager`?